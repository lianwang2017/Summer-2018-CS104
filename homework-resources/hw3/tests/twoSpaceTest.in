Student Athlete , COMM
Nisar Ahmed , EE
Aaron Cheney , CSCI