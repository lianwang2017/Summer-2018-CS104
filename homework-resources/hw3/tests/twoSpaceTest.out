EE 
Nisar Ahmed

CSCI 
Aaron Cheney

COMM 
Student Athlete

